implementation/jobberknoll/api/int/int-controller.ts

```diff
@@ -12,26 +12,32 @@ import {
 } from "./routes/mod.ts";
 
 export class IntController implements Controller {
-  public constructor(private readonly service: Service) {}
+  public constructor(
+    private readonly service: Service,
+    private readonly logger: Logger,
+  ) {}
 
   public get prefix(): string {
     return "/int/v1";
   }
 
   public get routes(): OpenAPIHono {
     const app = new OpenAPIHono({ defaultHook })
-      .openapi(getHealthRoute, getHealthHandler())
+      .openapi(
+        getHealthRoute,
+        getHealthHandler(this.service.getHealth),
+      )
       .openapi(
         getAccountByIdRoute,
         getAccountByIdHandler(this.service.getAccountById),
       );
 
-    configureErrorHandler(app);
+    configureErrorHandler(app, this.logger);
 
     configureDocs(app, {
       path: this.prefix,
       title: "Jobberknoll Internal API",
-      version: denoJson.version,
+      version: workspace.version,
```

Comment:
Consolidating separate versions for subpackages in the workspace to a single workspace version.